lib/MusicBrainz/Server/Edit/Alias/Delete.pm

```diff
@@ -8,7 +8,7 @@ use MusicBrainz::Server::Data::Utils qw( partial_date_to_hash );
 use MusicBrainz::Server::Edit::Types qw( Nullable PartialDateHash );
 
 extends 'MusicBrainz::Server::Edit';
-
+with 'MusicBrainz::Server::Edit::Alias';
```

Comment:
Only for completeness (in case we want to put more alias-specific common functions in there).